terraform/cross-device/variables.tf

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "environment" {
+  description = "Description for the environment, e.g. dev, staging, production."
+  type        = string
+  default     = "demo-dev"
+}
+
+variable "project_id" {
+  description = "GCP Project ID in which this module will be created."
+  type        = string
+}
+
+variable "region" {
+  description = "Region where all services will be created."
```

Comment:
services -> resources. Also, it's not guaranteed that we create "all" resources in said region (think about global services).

Reply:
Done